README.md

```diff
@@ -68,6 +68,8 @@ If you are planning to learn C# programming language then there are a plethora o
 
 [9) msdn channel ](https://mva.microsoft.com/en-us/training-courses/c-fundamentals-for-absolute-beginners-16169)
 
+[10) Scaler Topics](https://www.scaler.com/topics/c/)
```

Comment:
Thank you for your contribution!

This is a C tutorial, not C# and as a result, I think it should have it's own section. Something like this:

``` markdown
# C Tutorials:

[1) Scaler Topics](https://www.scaler.com/topics/c/)
```